Lib/test/libregrtest/run_workers.py

```diff
@@ -483,17 +501,25 @@ def __init__(self, num_workers: int, runtests: RunTests,
             self.worker_timeout: float | None = min(self.timeout * 1.5, self.timeout + 5 * 60)
         else:
             self.worker_timeout = None
-        self.workers: list[WorkerThread] | None = None
+        self._workers: list[WorkerThread] | None = None
 
```

Comment:
Maybe we could just initialise self.workers to an empty list here? Then we wouldn't need the wrapper property